.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,106 @@
+---
+name: Release issue template
+about: Tracking issue for new releases
+title: Manta {{ SET_VERSION }} Release checklist
+---
+# Release Checklist
+
+Most of the following checks should be completed before officially publishing the new release
+of the Calamari/Manta runtime or client. Some need to be completed after the new code is deployed.
+
+### Runtime Releases
+
+These checks should be performed on the codebase prior to forking to a release-
+candidate branch.
+
+- [ ] Verify [`spec_version`](#spec-version) has been incremented since the
+    last release for any native runtimes from any existing use on public
+    (non-private/test) networks. If the runtime was published (release or pre-release), either
+    the `spec_version` or `impl` must be bumped.
+- [ ] Verify pallet and [extrinsic ordering](#extrinsic-ordering) has stayed
+    the same. Bump `transaction_version` if not.
+- [ ] Verify new extrinsics have been correctly whitelisted/blacklisted
+- [ ] Verify [benchmarks](#benchmarks) have been updated for any modified
+    runtime logic.
+
+The following checks can be performed after we have forked off to the release branch.
+- [ ] Complete the following [manual QA workflow](https://hackmd.io/TbFmorG2RnOPmLuFcg9JOQ?view).
+- [ ] Verify [Polkadot JS API](#polkadot-js) are up to date with the latest
+    runtime changes.
+- [ ] Execute runtime upgrade to Como and verify network stability.
+- [ ] Execute runtime upgrade to Baikal and verify network stability.
+- [ ] Prepare a governance post and submit to our forum with description and motivation for changes.
+
+### All Releases
+
+- [ ] Check that the new client versions have [run on the network](#burn-in)
+    without issue for 12 hours.
+- [ ] Check that a draft release has been created at
+    https://github.com/Manta-Network/Manta/releases with relevant [release
+    notes](#release-notes)
+- [ ] Check that [build artifacts](#build-artifacts) have been added to the
+    draft-release
+- [ ] Coordinate with marketing team for documenation updates and other relevant tasks.
```

Comment:
Can you add a item here for updating changelog?

Reply:
Done.